plugin/vcs/util.go

```diff
@@ -74,6 +75,7 @@ func (p PushEvent) GetDistinctFileList() []DistinctFileItem {
 				Commit:    commit,
 				FileName:  fileName,
 				ItemType:  itemType,
+				IsYAML:    strings.HasSuffix(fileName, ".yml"),
```

Comment:
Is this case-sensitive?

Reply:
Yes, and I don't think it makes sense to support extensions like `.YML`, `.yML`, `.YmL` 😅

Reply:
That sounds good too, as long as we mention it in the document.